src/global/networkConstants.ts

```diff
@@ -394,6 +395,23 @@ export const chainProperties: types.ChainPropType = {
 		externalLinks: 'https://centrifuge.api.subscan.io',
 		rpcEndpoints: []
 	},
+	//TODO: Aleem=> Need to update collective network, currently using polkadot data
+	[network.COLLECTIVES]: {
```

Comment:
is this updated?

Reply:
@alphainfinitus Collectives will use polkadot config except rpc and subsquid url. Have pushed the updated ones